src/javascripts/helpers/router.js

```diff
@@ -0,0 +1,20 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import loginButton from '../components/buttons/loginButton';
+import logoutButton from '../components/buttons/logoutButton';
+import firebaseConfig from './auth/apiKeys';
+import startApp from '../views/startApp';
+
+const checkLoginStatus = () => {
+  firebase.initializeApp(firebaseConfig);
+  firebase.auth().onAuthStateChanged((user) => {
+    startApp(user);
```

Comment:
Tell me about this being outside of the conditional. There is nothing wrong with this of course, I just would like to know how you came to do this 😄

Reply:
It allows the app to start regardless of whether the user is logged in or not. That way any user can READ only if they are unauthenticated.

Reply:
OK! I love it! So you are conditionally showing the buttons?

Reply:
Yes, just like in the ternary you pointed out above!